src/components/Calendar/index.tsx

```diff
@@ -216,6 +219,62 @@ export default function Calendar({
           meetings.push(...friendMeetings);
         })
     );
+    Object.entries(versions)
```

Comment:
try to combine this loop with the one, since they're the exact same. The objects being iterated over can be combined into a single array maybe.